public_html/wp-content/plugins/wordcamp-speaker-feedback/wordcamp-speaker-feedback.php

```diff
@@ -25,4 +27,64 @@
 function load() {
 	require_once PLUGIN_DIR . 'includes/class-feedback.php';
 	require_once PLUGIN_DIR . 'includes/comment.php';
+	require_once PLUGIN_DIR . 'includes/form.php';
+	require_once PLUGIN_DIR . 'includes/page.php';
+}
+
+/**
+ * Create main Feedback page.
+ */
+function activate() {
+	add_feedback_page();
+}
```

Comment:
I'm wondering if this will work if this plugin ends up being network activated. I.e., when a new site is created, and this is already network activated, will it still run this and create the page on the new site?

Unfortunately, when I tried to test this on my Docker, I kept running into memory limits and max execution times, which is probably (hopefully) an unrelated problem, so I don't know the answer yet. I did notice that when I network activated the plugin, existing sites in the network did not have a feedback page added.

Reply:
Good point, I didn't test this network-wide.

Reply:
I've fixed it so that it will also check for the page on `init` and add it if necessary… do you think I should also update the deactivation cleanup to account for network deactivations? seems like that would probably time out on wc.org, looping through every site just to delete a page.

Reply:
> do you think I should also update the deactivation cleanup to account for network deactivations? seems like that would probably time out on wc.org, looping through every site just to delete a page.

Yeah, probably not worth it to worry about deactivation. This did make me think that we should probably add a skip feature flag here though, and only add the page on sites past a certain ID.

Reply:
> skip feature flag

should the entire plugin functionality be on a skip feature flag? or would we still want the `/session/blah/feedback` form to work on every site?

Reply:
Good point, it should probably just be the entire plugin.

Reply:
We can do that in a separate PR, then 👍